shallow_backup/reinstall.py

```diff
@@ -73,6 +73,12 @@ def reinstall_configs_sb(configs_path):
 		elif os.path.isfile(path_to_backup):
 			copyfile(path_to_backup, dest_path)
 
+	# reinstall crontab
+	with open("{0}/crontab.txt".format(configs_path), "r") as f:
```

Comment:
`os.path.join` here, as well. Also need to be careful not to overwrite existing crontabs. Let's make a backup of that before we overwrite it, or get user confirmation (that can be skipped with a flag or something)

Reply:
The used bash command doesn't override existing lines.